eslint/babel-eslint-parser/src/convert/convertTokens.js

```diff
@@ -1,4 +1,10 @@
-import { tokTypes as tt } from "@babel/core";
+import { tokTypes } from "@babel/core";
+
+const tl = (process.env.BABEL_8_BREAKING
```

Comment:
Do we want to try to figure out a way to test this? Would it work to just manually set `process.env.BABEL_8_BREAKING` in our tests?

Reply:
Yeah, it's tested by the "Test Babel 8 breaking changes" job (even if it doesn't show up in codecov)